src/applications/vaos/components/NeedHelp.jsx

```diff
@@ -18,13 +18,15 @@ export default function NeedHelp() {
       <p className="vads-u-margin-top--0">
         For help scheduling a VA or Community Care appointment, please call{' '}
         <a href="tel:8774705947">877-470-5947</a> (if you have hearing loss,
-        call <Telephone contact={CONTACTS['711']} pattern={PATTERNS['911']} />
+        call TTY:{' '}
+        <Telephone contact={CONTACTS['711']} pattern={PATTERNS['911']} />
```

Comment:
Is the pattern supposed to be 911 (rather than 711)?

Reply:
Yupp! Originally there wasn't the 711 number when the Telephone component was created just 911, which is why it's PATTERNS['911'] now. That should probably be updated in formation-react to be something like `PATTERNS.3_DIGITS` or something like that 🙂